atd-vzd/README.md

```diff
@@ -0,0 +1,51 @@
+## Vision Zero Database
+
+Currently the Vision Zero database consists of a Postgres 10 database that mirrors the CRIS public crash data format, with added field lookup tables.
+
+At the moment, only the schema, views and triggers are posted on here and version-controlled. The actual data can be gathered from the [CRIS website]([https://cris.dot.state.tx.us](https://cris.dot.state.tx.us/)). 
+
+Currently the schema is being tracked in the following folders:
+
+- `schema`- Contains the original table definitions, 
+- `migrations` - Contains sql modifications to the original files in schema, all timestamped with date and time in a `yyyy-mm-dd--hhmm` format.
```

Comment:
Since you have a README for migrations, and a user might not know to go look for those docs, could you add a link/reference to it here. 

Something simple like: `[More info about migrations in this README](./migrations/README.md)`